Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1665,3 +1669,63 @@ extension BrowserViewController: IntroViewControllerDelegate {
         })
     }
 }
+
+extension BrowserViewController: ContextMenuHelperDelegate {
+    func contextMenuHelper(contextMenuHelper: ContextMenuHelper, didLongPressElements elements: ContextMenuHelper.Elements) {
+        let actionSheetController = UIAlertController(title: nil, message: nil, preferredStyle: UIAlertControllerStyle.ActionSheet)
+        var dialogTitle: String?
+
+        if let url = elements.link {
+            dialogTitle = url.absoluteString
+            let newTabTitle = NSLocalizedString("Open In New Tab", comment: "Context menu item for opening a link in a new tab")
```

Comment:
People have also been asking for "Open in new Background Tab' I think. Should we add that also?